Description: Fix hyphen-used-as-minus-sign in the man page.
Forwarded: No
Author: Julián Moreno Patiño <darkjunix@gmail.com>
Last-Update: 2012-12-23
--- a/hydra.1
+++ b/hydra.1
@@ -79,7 +79,7 @@ tried on all logins, then the next passw
 exit after the first found login/password pair (per host if \-M)
 .TP
 .B \-F
-exit after the first found login/password pair for any host (for usage with -M)
+exit after the first found login/password pair for any host (for usage with \-M)
 .TP
 .B \-M FILE
 server list for parallel attacks, one entry per line
